api/test/app.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var BitcoreHTTP = require('../lib/http');
+
+module.exports = function(nodeMock) {
+  return process.env.INTEGRATION === 'true' ? BitcoreHTTP.create().app : new BitcoreHTTP(nodeMock).app;
```

Comment:
Please make this a config and just check that `process.env.INTEGRATION` is just not "falsy". If I'm not familiar enough and do `export INTEGRATION=1` I'd like this to work.